tests/yam/validate/validate_migration_logs.pm

```diff
@@ -21,10 +21,16 @@ sub run {
     script_run("tar czvf /tmp/udev_rules.tar.gz /etc/udev/rules.d/*", {timeout => 60});
     upload_logs("/tmp/udev_rules.tar.gz", failok => 1);
 
-    if (script_run("cat /var/log/distro_migration.log | grep -i -E \"migration failed|aborting migration\"") == 0) {
-        record_info("Migration failed", script_output("cat /var/log/distro_migration.log | grep -i -E \"migration failed|aborting migration\" -B50"), result => 'fail');
+    # fatal migration error
```

Comment:
now with the variables you don't need the comment  here or below, doesn't provide any additional information.

Reply:
Done